docs/source/usage/python.rst

```diff
@@ -233,6 +233,10 @@ Collective Effects & Overall Simulation Parameters
 
       Run the envelope tracking simulation loop.
 
+   .. py:method:: track_reference()
```

Comment:
Opps, `ref` argument missing.

Reply:
Will push a fix

Reply:
Oh, yes I ran this, but I missed that the argument was missing in the doc here.

Reply:
I changed the API after the first doc draft and forgot to add this. Fixed now via dfe55da5c4a0344d42675c20ef6fd018242dcd94 :)